src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/DraftApprenticeshipController.cs

```diff
@@ -773,5 +788,62 @@ private async Task<ValidateUlnOverlapOnStartDateQueryResult> HasStartDateOverlap
                 model.EndDate.Date.Value.ToString("dd-MM-yyyy")
             );
         }
+
+        private async Task<IActionResult> HandleLearnerDataSync(EditDraftApprenticeshipViewModel model)
+        {
+            try
+            {
+                var response = await _outerApiService.SyncLearnerData(model.ProviderId, model.CohortId.Value, model.DraftApprenticeshipId.Value);
+                
+                if (response.Success)
+                {
+                    TempData.Put(LearnerDataSyncDraftApprenticeshipKey, response.UpdatedDraftApprenticeship);
+                    TempData[LearnerDataSyncSuccessKey] = LearnerDataSyncSuccessMessage;
+                }
+                else
+                {
+                    TempData[LearnerDataSyncErrorKey] = response.Message ?? LearnerDataSyncErrorMessage;
+                }
+            }
+            catch (Exception e)
+            {
+                TempData[LearnerDataSyncErrorKey] = LearnerDataSyncExceptionMessage;
+            }
+
+            return RedirectToAction("EditDraftApprenticeship", "DraftApprenticeship", model);
```

Comment:
There is bit of a side affect when you try to do this. The URL gets populated with the full model

`...YGNRKK/edit?DraftApprenticeshipId=190342&CohortId=150172&AccountLegalEntityId=0&DateOfBirth=SFA.DAS.CommitmentsV2.Shared.Models.DateModel&StartDate=SFA.DAS.CommitmentsV2.Shared.Models.Month....`